### ISSUE TITLE
Add feedback buttons in the detail page of a scanner result

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-server#13078]
We should be able to report feedback on a scanner result in the "detail" page.

**Comment by ioanarusiczki** [comment]
@willdurand 
- on AMO -dev I did not find problems marking as `false positive` or `true positive` an add-on from it's `change` form. 
- the action buttons are available for `admin/models/scanners/scannerresult/` or in the `change` page only for the add-ons which contain matched rules. 

As you[ said ](https://github.com/mozilla/addons-server/pull/13080) when I mark an add-on as `false positive` the `change` redirects to the general models/scanners/scannerresult/ page and a message highlighted in green is displayed at the top of the page 
`Scanner result 5423 has been marked as false positive.` 
Meanwhile, the 404 page (because I have no access to the github repo) opens in a new tab. 


- On AMO stage - when I try to mark an add-on as false positive instead of the 404 for the Github repo I receive a 502 Bad Gateway. The general scanner result page is displayed but without a green message at the top of the page `Scanner result 5423 has been marked as false positive.`

**Comment by willdurand** [comment]
> On AMO stage - when I try to mark an add-on as false positive instead of the 404 for the Github repo I receive a 502 Bad Gateway. The general scanner result page is displayed but without a green message at the top of the page `Scanner result 5423 has been marked as false positive.`

mmmh the 502 could be a problem with the length of the results if, for example, the yara rule matches all the files (and if there are lots of files in the add-on).

**Comment by ioanarusiczki** [comment]
@willdurand Yes, I could report on stage false positive without the 502 for this result `Scanner result 1446 has been marked as false positive.`  It's `rules-for-stage (version: 1.22.2)`
Looks like the add-on `many-files-stage (version: 1.22.2)` is having this issue. It's been uploaded over and over again.

**Comment by AlexandraMoga** [comment]
@willdurand it looks like the 502 error does indeed happen only if the following two conditions are met: 
1. the add-on has many files
2. the add-on matches many rules (4 `yara` rules in this case)

One of the scanner results that has this issue is [1654](https://addons-internal.stage.mozaws.net/en-US/admin/models/scanners/scannerresult/1654/change/)

If you report a false positive for a `customs` result of the same add-on, the 502 doesn't happen because, I'm assuming, you fixed this in  #13061

In conclusion, this does seem like a rare or unlikely scenario, same as https://github.com/mozilla/addons-server/issues/13037#issuecomment-562588047
